toolchest_client/api/query.py

```diff
@@ -172,6 +172,7 @@ def run_query(self, tool_name, tool_version, input_prefix_mapping,
         self._upload_docker_image(custom_docker_image_id)
         self._check_if_should_terminate()
 
+        self._update_status(Status.TRANSFERRED_FROM_CLIENT)
```

Comment:
Nit: This might be better on L173 before the  `chech_if_should_terminate`, so the connection with the upload is more apparent.